warehouse/query-core/src/test/java/datawave/query/AnyFieldQueryTest.java

```diff
@@ -283,15 +283,15 @@ public void testOrOr() throws Exception {
             anyQuery += JEXL_OR_OP + CityField.STATE.name() + statePhrase;
             anyQuery += JEXL_OR_OP + "_NOFIELD_" + contPhrase;
             String plan = getPlan(query, true, true);
-            String expected = anyQuery.replace(JEXL_OR_OP + "_NOFIELD_" + contPhrase, "");
-            assertPlanEquals(expected, plan);
+            assertPlanEquals(anyQuery, plan);
```

Comment:
@FineAndDandy Why is the _NOFIELD_ expression no longer getting replaced with a false?

Reply:
@FineAndDandy It appears I was premature with my review..... sorry.